assets/js/leadership-team.js

```diff
@@ -0,0 +1,102 @@
+const leaders = [
+  {
+    captains: [
+      {
+        name: "Yan-Yin Choy",
+        image: "../img/leadership/captains/yan-yin.png",
+        link: "http://linkedin.com/in/yanyinchoy",
+      },
+      {
+        name: "Joey Richardson",
+        image: "../img/leadership/captains/joey.png",
+        link: "https://www.linkedin.com/in/joseph-richardson-97206953/",
+      },
+      {
+        name: "Annie Steenson",
+        image: "../img/leadership/captains/annie.png",
+        link: "http://linkedin.com/in/anniesteenson",
+      },
+    ],
+  },
+  {
+    leader: [
+      {
+        name: "Cesar Arellano",
+        image: "../img/leadership/team/cesar.png",
+        link: "https://www.linkedin.com/in/cesaarel",
+      },
+      {
+        name: "Diane Labenz",
+        image: "../img/leadership/team/diane.png",
+        link: "https://www.linkedin.com/in/dianelabenz/",
+      },
+      {
+        name: "Minh Nguyễn",
+        image: "../img/leadership/team/minh.jpg",
+        link: "http://notes.1ec5.org/",
+      },
+      {
+        name: "Dane Olsen",
+        image: "../img/leadership/team/dane.png",
+        link: "https://www.linkedin.com/in/danekolsen/",
+      },
+      {
+        name: "Lena Tran",
+        image: "../img/leadership/team/lena.png",
+        link: "https://www.linkedin.com/in/lenatran4/",
+      },
+      {
+        name: "Kate Valdes",
+        image: "../img/leadership/team/kate.png",
+        link: "https://katevaldes.com/portfolio/",
+      },
+    ],
+  },
+  {
+    advisory: [
+      { name: "Lorin Camargo", image: false, link: "#" },
+      { name: "Namrata Challa", image: false, link: "#" },
+      { name: "Erika Hudiono", image: false, link: "#" },
+      { name: "Sunny Mui", image: false, link: "#" },
+      { name: "Darren Pham", image: false, link: "#" },
```

Comment:
Headshot:
https://github.com/darpham/Basic-Portfolio.io/blob/master/assets/images/darren.pham.jpeg

LinkedIn: https://www.linkedin.com/in/darep-h20/

Reply:
Added.